website/docs/reference/insights.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Insights
+---
+
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+:::info Availability
+
+Insights is released in Unleash 6.0 and is available for Enterprise and Pro customers. Pro customers only see the first two charts.
+
+:::
+
+Insights is a feature designed to help you better understand and gain insights into what is happening in your Unleash instance. You can view insights across all projects or by selecting single or multiple projects using the filter.
+
+In total, there are 6 different charts available that show information over time:
+
+- Total users
+- Total flags
+- Average health
+- Median time to production
+- Flag evaluation metrics
+- Updates per environment type
```

Comment:
I know we say "the first two charts" further up, but could perhaps add (Pro, Enterprise) and (Enterprise) to the chart options available to each plan?